docs/standard/library-guidance/index.md

```diff
@@ -23,18 +23,20 @@ Aspects of high-quality open-source .NET libraries:
 
 ## Recommendations
 
-With each article, there is a list of recommendations for your open-source .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed. For example, a **Do** recommendation is one that should almost always be followed:
+With each article, there is a list of recommendations for your .NET library using **Do**, **Consider**, **Avoid**, and **Do not**. The wording of each recommendation indicates how strongly it should be followed.
+
+A **Do** recommendation is one that should almost always be followed:
 
 **✔️ DO** distribute your library using a NuGet package.
 
 On the other hand, **Consider** recommendations should generally be followed, but there are legitimate exceptions to the rule and you shouldn't feel bad about not following the guidance:
 
 **✔️ CONSIDER** using [SemVer 2.0.0](https://semver.org/) to version your NuGet package.
 
-**Do not** indicates something you should almost never do:
+**Avoid** recommendations are things that aren't a good idea, but breaking the rule sometimes makes sense:
```

Comment:
this is somewhat awkward. "...are generally not a good idea, but breaking..."?

Reply:
Is that better?